tests/services-lorri.nix

```diff
@@ -0,0 +1,24 @@
+{ config, pkgs, ... }:
+
+let
+  lorri = pkgs.runCommand "lorri-0.0.0" {} "mkdir $out";
+  plistPath = "${config.out}/user/Library/LaunchAgents/org.nixos.lorri.plist";
```

Comment:
@LnL7 To make plist testing less fragile, we could use `plutil` which can lint, read and convert plist files to json. How could I integrate it into the tests here?

Reply:
That's a great idea, we have plutil as part of `xcbuild` so that should work.